client/components/progress-bar/index.jsx

```diff
@@ -35,7 +39,9 @@ module.exports = React.createClass( {
 
 	render: function() {
 		return (
-			<div className={ classnames( this.props.className, 'progress-bar' ) }>
+			<div className={ classnames( this.props.className, 'progress-bar', {
+				'is-compact': this.props.isCompact
+			} ) }>
```

Comment:
The multi-line opening tag is a bit hard to read. Would you mind assigning a `className` or `classes` variable separately at the top of the function?